packages/sdk/src/adapters/websocket/adapter.ts

```diff
@@ -31,69 +24,40 @@ export type WebSocketAdapterOptions = AdapterOptions;
  *  - Server-based multiuser topologies
  */
 export class WebSocketAdapter extends Adapter {
-	/**
-	 * Creates a new instance of the WebSocket Adapter.
-	 */
-	constructor(options?: WebSocketAdapterOptions) {
-		super(options);
-	}
 
-	/**
-	 * Start the adapter listening for new connections.
-	 * @param onNewConnection Handler for new connections.
-	 */
-	public listen() {
-		if (!this.server) {
-			// If necessary, create a new web server.
-			return new Promise<Restify.Server>((resolve) => {
-				const server = this.server = Restify.createServer({ name: "WebSocket Adapter" });
-				this.server.listen(this.port, () => {
-					this.startListening();
-					resolve(server);
-				});
-			});
-		} else {
-			// Already have a server, so just start listening.
-			this.startListening();
-			return Promise.resolve(this.server);
-		}
-	}
-
-	private startListening() {
-		// Create a server for upgrading HTTP connections to WebSockets.
-		const wss = new WS.Server({ server: this.server, verifyClient });
+	/** @inheritdoc */
+	public get name(): string { return this. constructor.name; }

```

Comment:
Is there a space here? Does that compile?

Reply:
Weird. Yep it compiles. Will fix.

---
In reply to: [332696471](https://github.com/microsoft/mixed-reality-extension-sdk/pull/453#discussion_r332696471) [](ancestors = 332696471)